index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <title>Document</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <h2 class="nav-name">Mark Fenekayas</h2>
+        <div class="nav-items">
+            <h4 class="items-click">Portfolio</h4>
+            <h4 class="items-click">About</h4>
+            <h4 class="items-click">Contact</h4>
+            <i class="fa-regular fa-envelope items-click items-icon"></i>
+        </div>
+    </nav>
+
+    <section class="first-section">
+        <section class="left-section">
+            <div class="contact-icons">
+                <i class="fa-brands fa-github"></i>
+                <i class="fa-brands fa-linkedin-in"></i>
+                <i class="fa-brands fa-angellist"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <i class="fa-solid fa-m"></i>
+            </div>
```

Comment:
- [ ] It's better to change the social media icons wrapped in div with unorder list using 'ul` and `li` for better user experience UX and semantic reasons.

Example
```
<ul>
  <li>github</li>
  <li>linkedIn</li>
  <li>twitter</li>
</ul>
```